server/views/pages/overview/partials/timelineTab/timelineTabContents.njk

```diff
@@ -3,25 +3,26 @@
 
     <div class="moj-timeline">
 
-      <!-- Timeline item -->
+      {% for event in timeline.events %}
+        <div class="moj-timeline__item">
+          <div class="moj-timeline__header">
+            <h2 class="moj-timeline__title">{{ event.eventType | formatTimelineEvent }}</h2>
+            <p class="moj-timeline__byline govuk-!-display-none-print">by {{ event.actionedByDisplayName }}, {{ event.prisonId }}</p>
+          </div>
+
+          <p class="moj-timeline__date">
+            <time datetime="{{ event.timestamp }}">{{ event.timestamp | formatDate('D MMMM YYYY') }}</time>
+          </p>
+
+          {# TODO: Display the description and action
```

Comment:
We need to figure out the logic for when to display a "description" with an "action" (eg. Link, Tag) etc